src/common/SPELLS/demonhunter.ts

```diff
@@ -153,11 +153,16 @@ const spells = {
     name: 'Immolation Aura',
     icon: 'ability_demonhunter_immolation',
   },
-  IMMOLATION_AURA_FIRST_STRIKE: {
+  IMMOLATION_AURA_FIRST_STRIKE_DPS: {
```

Comment:
Used to be "IMMOLATION_AURA_FIRST_STRIKE_DPS" when looking at old comments, fits more that way IMO.

Reply:
What does "First Strike" mean here? Is it the damage on use, or?

Reply:
This is the first stike (aoe dmg), it does directly when you activate.
![image](https://user-images.githubusercontent.com/4808216/103044331-6c4c4400-4580-11eb-8dcd-85644c256f68.png)

-> this is the buff which does aoe dmg after the strike
![image](https://user-images.githubusercontent.com/4808216/103044404-a3225a00-4580-11eb-8610-850a340cae67.png)

Reply:
I feel like IMMOLATION_AURA_INITIAL_HIT is more telling, and use DAMAGE instead of DPS imo.

Reply:
Hit implies that two things are making contact, I would say this is more a strike.
I agree on the DAMAGE instead of DPS tho.

Reply:
I want to add to this, I believe it was called dps because both havoc and veng could have this spell but the first strike spell Id was different based on spec. Something you want to check. Probably should have been called havoc instead of dps.

Reply:
Looked up what they actually do and we dont have to seperate them with _HAVOC and _VENGENCE anymore. Only the last one for vengence.

_From combat log_
HAVOC
**258920 - first cast
258921 - initial hit
258922 - buff aoe**

VENGENCE
**258920 - first cast
258921 - initial hit
258922 - buff aoe**
320334 - (talent)infernal armor, extra dmg vs melee

We could go with your initial_hit,
```
IMMOLATION_AURA
IMMOLATION_AURA_INITIAL_HIT
IMMOLATION_AURA_BUFF
```

Do we still add _DAMAGE on the initial and buff?

Reply:
I would prefer the first one with _damage

Reply:
Changed it to this
```
IMMOLATION_AURA
IMMOLATION_AURA_INITIAL_HIT_DAMAGE
IMMOLATION_AURA_BUFF_DAMAGE
```